app/models/ride.rb

```diff
@@ -172,14 +170,30 @@ def self.to_csv
       end
     end
   end
-  
+
+  def has_conversation_with_messages?
+    !! conversation.present? && conversation.messages.present?
+  end
+
+  def break_out_city_state( direction )
+    city_state = self.send("#{direction}_city_state")
+    if city_state.present? && city_state.include?(',')
+      city_state_array = self.send("#{direction}_city_state").split(',')
+      self.send("#{direction}_city=", city_state_array[0].try(:strip))
+      self.send("#{direction}_state=", city_state_array[1].try(:strip))
+    end
+  end
+
   def email
     self.voter&.email
   end
-  
-  def phone_number
-    self.voter&.phone_number
-  end
+
+  # Commented this out because it's conflicting with the attr_accessor above
+  # the attr_accessor should only be used as a passthrough, if you need the phone
+  # number use ride.voter.phone_number directly
+  # def phone_number
+  #   self.voter&.phone_number
+  # end
```

Comment:
What do you think of
```rb
def phone_number
  voter&.phone_number
end

def phone_number=(number)
  raise "Cannot set phone_number on a voterless ride" unless voter
  voter.phone_number = number
end
```
That way, the getter and setter stay in-sync with one another (what this PR fixes) _and_ with the voter (which this PR breaks).